src/cmds/pbs_rsub.c

```diff
@@ -792,6 +816,9 @@ main(int argc, char *argv[], char *envp[])
 		for (; *hostp; hostp++) {
 			int host_ncpus = 0;
 
+			if (strlen(*hostp) == 0)
+				continue;
```

Comment:
@vchlum, I am not sure, if we really need this check.

Reply:
It handles the special case 'pbs_rsub ... --hosts ""'. It can happen e.g. within the PTL test. I suggest to handle it instead of changing the test.

Reply:
Ok. Then, I believe we should handle this in process_opts()?